frontend/src/components/common/Filter/components/FilterContent/components/common/style.ts

```diff
@@ -73,7 +71,7 @@ export const listItemButtonSelectedCss = (props: CommonThemeProps) => {
 
       .MuiListItemText-root {
         span:first-of-type {
-          font-weight: ${semibold(props)};
+          font-weight: ${fontWeightSemibold(props)};
```

Comment:
we need to explicitly call `fontWeightSemibold(props)` here, because it's using `css from "@emotion/react"`